app/i18n/translations/zh-Hant-TW.json

```
@@ -1,4 +1,10 @@
 {
+  "closeApp": "結束 App",
+  "initializing": "初始化中",
```

```suggestion
  "initializing": "起動中",
```
`初始化中` 感覺很機械， `起動中` 會否較好？

---

or 啓動中